src/add_remove_todo.js

```diff
@@ -0,0 +1,71 @@
+const myList = document.querySelector('.todo-list');
+const todoBtn = document.querySelector('.todo-btn');
+const todoInput = document.querySelector('.todo-input');
+
```

Comment:
- [ ] Please fix this issue that the user face when he try to add or remove items
![image](https://user-images.githubusercontent.com/74744787/179365334-e012868f-f5b2-4674-8ee9-06a322ff4494.png)

Reply:
Hello @MeqdamAlqudah, 
Thank you for your review. Please, from my end, everything is working fine  as I can add and delete. Kindly, let me know what to be done.  Thank you.
![image](https://user-images.githubusercontent.com/101729024/179367816-d51e2842-f0b7-4567-ad08-82eeaa17aba2.png)

Reply:
@ecekpo  try to request another review if you didn't face the issue